src/eddington/fit_functions_list.py

```diff
@@ -248,3 +249,32 @@ def normal(a, x):
     :return: float
     """
     return a[0] * np.exp(-(((x - a[1]) / a[2]) ** 2)) + a[3]
+
+
+@fit_function(
+    n=3,
+    syntax="a[0] * (a[1] ** x) * exp(-a[1]) / gamma(x+1) + a[2]",
+    x_derivative=lambda a, x: (
+        a[0] * (a[1] ** x) * np.exp(-a[1]) / scipy.special.gamma(x + 1)
+    )  # noqa: W503
+    * (np.log(a[1]) - scipy.special.digamma(x + 1)),  # noqa: W503
+    a_derivative=lambda a, x: np.stack(
+        [
+            (a[1] ** x) * np.exp(-a[1]) / scipy.special.gamma(x + 1),
+            a[0]
+            * np.exp(-a[1])  # noqa: W503
+            / scipy.special.gamma(x + 1)  # noqa: W503
+            * (x * a[1] ** (x - 1) - a[1] ** x),  # noqa: W503
+            np.ones(shape=np.shape(x)),
+        ]
+    ),
+)  # pylint: disable=C0103
+def poisson(a, x):
+    """
+    Poisson fit function.
+
+    :param a: Coefficients array of length 3
```

Comment:
Pay attention that there are 4 parameters, not 3 :)

Reply:
Why 4? a[0], a[1] and a[2]

Reply:
You are right, my bad.